travis_configure.sh

```diff
@@ -104,13 +131,20 @@ run_tests()
             print_error_msg "Invalid build configuration"
             return 1
         fi
-
+    elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        autoreconf -fi
+        # Pass the root of the openssl installation directory
+        ./configure --with-openssl=$(brew --prefix openssl)
+        make check
+    else
+        print_error_msg "Invalid build configuration"
+        return 1
     fi
 
     return 0
 }
 
-## Script entrypoint
+# Script entrypoint
```

Comment:
Nitpicking: this change better goes into the minor formatting commit rather than this commit